src/server/lib/okta/idx/challenge.ts

```diff
@@ -287,104 +275,6 @@ export const challengeResend = (
 	});
 };
 
-/**
- * @name setPasswordAndRedirect
- * @description Okta IDX API/Interaction Code flow - Answer a challenge with a password, and redirect the user to set a global session and then back to the app. This could be one the final possible steps in the authentication process.
- * @param stateHandle - The state handle from the previous step
- * @param password - The password to set
- * @param expressReq - The express request object
- * @param expressRes - The express response object
- * @param introspectRemediation - The remediation object name to validate the introspect response against
- * @param path - The path of the page
- * @param ip - The ip address
- * @returns Promise<void> - Performs a express redirect
- */
-export const setPasswordAndRedirect = async ({
```

Comment:
This function was moved to `submitPasscode.ts` as it was causing circular dependencies, causing jest to error with `ReferenceError: Cannot access 'x' before initialization`

It doesn't make sense for this function to be in the challenge file anyway!